model/testresult/testresult.go

```diff
@@ -162,14 +162,14 @@ func TestResultsQuery(taskIds []string, testId, testName, status string, limit,
 }
 
 // TestResultsFilterSortPaginate is a query for returning test results to the taskTests GQL Query.
-func TestResultsFilterSortPaginate(taskIds []string, testName, status, sortBy string, sortDir, page, limit, execution int) ([]TestResult, error) {
+func TestResultsFilterSortPaginate(taskIds []string, testName string, statuses []string, sortBy string, sortDir, page, limit, execution int) ([]TestResult, error) {
 	tests := []TestResult{}
 	match := bson.M{
 		TaskIDKey:    bson.M{"$in": taskIds},
 		ExecutionKey: execution,
 	}
-	if len(status) > 0 {
-		match[StatusKey] = status
+	if len(statuses) > 0 {
+		match[StatusKey] = bson.M{"$in": statuses}
 	}
 
 	pipeline := []bson.M{
```

Comment:
@john-m-liu asked me to refactor my db query function for patch tasks to change it from using a pipeline aggregation to being a regular query. it has performance benefits i think. this could be a good time to do the same to this function since you're making changes to it. you'd basically do the same exact thing that i've done here: https://github.com/evergreen-ci/evergreen/blob/master/model/task/task.go#L1941

Reply:
Yep, it's worth creating a ticket to change it but doesn't need to be done in this PR since it wasn't introduced here

Reply:
I'll create a ticket for this. I remember when I tried this before (and just now), the duration field from the projection isn't available for the sort.

Reply:
https://jira.mongodb.org/browse/EVG-7584